### PR TITLE
fix error of exercise 13: min -> Float.min

### PR DESCRIPTION
**Pull request by beizhedenglong**

### CODE REVIEW

**Comment by yminsky** [comment]
A DCO is needed to submit! Can you fix that?

**Comment by beizhedenglong** [comment]
@yminsky fixed~

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 _build
 .merlin
 .DS_Store
+node_modules
```
Comment (yminsky) [inline .gitignore]: Why is there a change in the gitignore here?

Comment (beizhedenglong) [reply .gitignore]: I'm using a vs code  plugin([reason-language-server](https://github.com/jaredly/reason-language-server)) that support OCaml and ReasonMl. That plugin uses node_modules to store some information. Keep it maybe benefit many other vs code users or remove it?

Comment (yminsky) [reply .gitignore]: Got it.

**Pull request merged**